src/common/defines.h

```diff
@@ -2206,6 +2206,28 @@
 #define ROW_LAG			8
 #define COL_LAG			0       /* mini lag-o-meter */
 
+#define ROW_RESIST_FIRE		23
+#define COL_RESIST_FIRE		1
+
+#define ROW_RESIST_COLD		23
+#define COL_RESIST_COLD		2
+
+#define ROW_RESIST_ELEC		23
+#define COL_RESIST_ELEC		3
+
+#define ROW_RESIST_ACID		23
+#define COL_RESIST_ACID		4
+
+#define ROW_RESIST_POIS		23
+#define COL_RESIST_POIS		5
+
+#define ROW_RESIST_MANA		23
+#define COL_RESIST_MANA		6
+
+#define ROW_TEMP_ESP		25
```

Comment:
Indentation consistency?

Reply:
Tried to fixe this in CLion, Vim and github online editor, all of them shows that `25` on the same columt with other values

Reply:
That's wierd, but you're right. When I look at the [whole file](https://github.com/TomenetGame/tomenet/blob/e8fcb013e0297dad3f39141cddeb686559e299f3/src/common/defines.h#L2224-L2227), the indetation is ok.